roles/bootstrap-os/tasks/main.yml

```diff
@@ -30,7 +30,7 @@
 
 - name: Create remote_tmp for it is used by another module
   file:
-    path: "{{ lookup('config', 'DEFAULT_REMOTE_TMP', on_missing='skip', wantlist=True) | first | default('~/.ansible/tmp') }}"
+    path: "{{ ansible_remote_tmp | default('~/.ansible/tmp') }}"
```

Comment:
I'm pretty ok with this but I don't know what will happen for people who have old ansible versions and used remote_tmp setting in ansible.cfg...

Reply:
May be I keep the old task and make it non-fatal? and add a new task using `ansible_remote_tmp` as above? not sure what else we can do here

Reply:
I checked when `ansible_remote_tmp` was introduced and apparently it goes back to versions prior to the minimal ansible version required in kubespray (https://github.com/ansible/ansible/commit/84367eacef9824a24e3f3d9ffe75803732b38684) .
So I guess we're fine as is